### DISCUSSION TITLE
Mailchimp (or other) sign-up field on checkout

### DISCUSSION THREAD

**Discussion by emrysgreen** [duncanmcclean/simple-commerce#915] [Feature Requests]
It would be great to be able to have a tickbox or similar confirmation to also join a newsletter (or even the function to agree anything else) that we could hook in to and provide an action; such as then being able to use the users' email address (or other completed fields) to register in newsletter software during the check out process.

**Comment by duncanmcclean** [comment]
I don't think this will be something that'll be built into Simple Commerce, in an aim to keep Simple Commerce *simple*. 

However, you can build this yourself by listening to the [events](https://simple-commerce.duncanmcclean.com/events) Simple Commerce dispatches, then adding to Mailchimp from there. You're more than welcome to publish it on the Marketplace for others who may be looking for similar functionality.
